src/styles/calendar.scss

```diff
@@ -0,0 +1,34 @@
+@import "./mixins";
+@import "./variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: $calendar-width;
+
+  &_cell {
+    @include square($cell-size);
+    background-color: $cell-background-color;
+
+    box-sizing: border-box;
+    border: $cell-border;
+    transition: all 0.5s linear;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223090964-09aad621-e179-4c92-9c6b-e51ea4b89f1a.png)

also, don't forget to create variable for $transitionDuration
```suggestion
    transition: transform $transitionDuration linear;
```